llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -5197,12 +5197,8 @@ bool CombinerHelper::matchUDivByConst(MachineInstr &MI) {
       return false;
   }
 
-  auto CheckEltValue = [&](const Constant *C) {
-    if (auto *CI = dyn_cast_or_null<ConstantInt>(C))
-      return !CI->isZero();
-    return false;
-  };
-  return matchUnaryPredicate(MRI, RHS, CheckEltValue);
+  return matchUnaryPredicate(
```

Comment:
If you're going to put GlobalISel code in the patch, then you need to add more than `[SelectionDAG]` to the PR name.